Arch.tex

```diff
@@ -639,9 +641,16 @@ \subsection{CubeDM - Draft}
 This Cube Data Model (WD) presents an abstracted representation of N-Dimensional cube datasets and 
 serves as a framework on which to construct models for more specialized Astronomical datasets. 
 
-\subsection{ObsLocTAP} 
+\subsection{Mango - Draft}
```

Comment:
Mano doesn't belong in this version of the document. The Transform and DatasetDM drafts were included because they are part of the greater work scope that is partially in PR (to provide some context)... but otherwise drafts don't belong here.

Reply:
I do not understand this point. 
- Mango is  not less advanced than e.g. Transform or Cube. 
- It is a draft based on Meas and Coords
- It has been used for most of the workshop use cases. 
The unfortunate reality is that the _greater work scope_ you are talking about is hardly in PR. Only Meas and Coords are in PR, but they are so much questioned that we had to run a workshop to save them.

Reply:
I wasn't meaning to judge any of the drafts as more or less advanced, and I think originally had just included Transform because of it's tight coupling to Meas and Coords. I think it better to remove drafts rather than draw a line in the gray area.

Reply:
IMO, the area turns grey when some drafts are mentioned and some other are not without clear reason.
I won't argue more, you are the editor  anyway.